en/flight_controller/intel_aero.md

```diff
@@ -28,9 +28,69 @@ OS_VERSION = Poky Aero (Intel Aero Linux Distro) 1.5.1-dev (pyro)"
 AIRMAP_VERSION = 1.8
 FPGA_VERSION = 0xc1
 ```
+## Setup Intel Aero using Ubuntu
 
+This chapter is based on the [Intel Aero wiki](https://github.com/intel-aero/meta-intel-aero/wiki/90-%28References%29-OS-user-Installation). Follow the steps to install the OS and Intel Aero repository (The Yocto does not necessarily have to be upgraded first). The required material for those steps consists of:
 
-## Flashing
+1. Power supply (battery or network cable)
+1. Micro HDMI to HDMI cable to attach a monitor
+1. Micro USB3 to USB2 female adapter
+1. USB Hub to attach mouse and keyboard
+
+As soon as the steps under [Intel Aero repository](https://github.com/intel-aero/meta-intel-aero/wiki/90-%28References%29-OS-user-Installation#intel-aero-repository) are completed an aero kernel is installed. From there on make sure to always boot using this kernel.
+
+Follow the instructions to flash the BIOS, FPGA and Flight Controller. Edit the mavlink router config file to include the laptop IP as a UDP Endpoint. After all those steps are completed, the drone should automatically connect to the QGroundControl running on the laptop.
+
+In a next step ROS needs to be installed. Instructions on how to install ROS can be found [here](https://github.com/intel-aero/meta-intel-aero/wiki/05-Autonomous-drone-programming-with-ROS).
```

Comment:
In a next step ROS needs to be installed. 
TO 
ROS must to be installed in a following step.